imports/ui/views/organizations/AboutUsMission/AboutUsImages.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+
+class AboutUsImages extends Component {
+  render() {
+    return (
+      <div style={styles.images}>
+        {this.props.images.map((image, index) =>
```

Comment:
this may be more of a preference thing, but maybe move the mapping logic outside of the return but still within the render? May make it a little cleaner/easier to read?

ex: similar to how it is done in the Skills component?
`render() {`
`let images = this.props.images.map((stuff) => {return <div>more stuff</div>})`
`return (<div>{images}</div>)`